app/src/main/java/com/blackboxteam/app/blackbox/LoginScreenActivity.java

```diff
@@ -0,0 +1,82 @@
+package com.blackboxteam.app.blackbox;
+
+import android.content.Intent;
+import android.support.design.widget.Snackbar;
+import android.support.design.widget.TextInputLayout;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Toast;
+
+import com.blackboxteam.app.blackbox.util.Validation;
+
+import static com.blackboxteam.app.blackbox.util.Validation.validateEmail;
+import static com.blackboxteam.app.blackbox.util.Validation.validateMinLength;
+
+public class LoginScreenActivity extends AppCompatActivity implements View.OnClickListener {
+
+    Button loginBtn;
+    EditText etEmail, etPassword;
```

Comment:
Nitpick but I think your properties could benefit from more descriptive and consistent names. `loginButton` instead of `loginBtn`, `emailEditText` instead of `etEmail`. Try not to abbreviate too much, readability should come first. :)